app/controllers/api/movies_controller.rb

```diff
@@ -3,11 +3,24 @@ def index
     netflix = Rails.application.credentials.unogsng_key
 
     begin
-      response = RestClient.get('https://unogsng.p.rapidapi.com/search?type=movie&orderby=rating',
-                                headers = { 'x-rapidapi-key': netflix })
-      results = JSON.parse(response)['results'].select { |film| film['avgrating'] > 4 }
-      netflix_sorted = results.sort_by { |film| film['avgrating'] }.reverse
-      render json: { body: netflix_sorted[0..9] }, status: 200
+      if params.has_key?(:lat) && params.has_key?(:lon)
```

Comment:
This has_key thing. would you like to explain to me how it works/dose?

Reply:
it bascially acts as a boolean, it checks the params sent with the request. If Lat and Lon are present then returns true and continues in that if statement. If false continues to the elsif statement.

Reply:
👍👍